pkg/core/storage/memcached_store.go

```diff
@@ -340,6 +341,41 @@ func (s *MemCachedStore) PersistSync() (int, error) {
 	return s.persist(true)
 }
 
+// PersistPrivate flushes the content of every private into MemCachedStore s
+// supposedly that s is an underlying persistent store of every private.
+// MemCachedStore remains accessible for the most part of this action
+// (any new changes will be cached in memory) as far as every private.
+// However, PersistPrivate relies on fact that every private is a private
+// wrapper around s and every private will be disposed after persist, so further
+// usages of private is no-op.
+func (s *MemCachedStore) PersistPrivate(private ...*MemCachedStore) int {
+	var keys int
+	for i, p := range private {
+		// @roman-khimov, these checks are here as a protection from possible invalid usages of PersistPrivate that may happen in future,
+		// we may safely remove them to same some execution time. Remove?
+		if !p.private {
+			panic(fmt.Sprintf("memcached store %d is not private", i)) // program bug
+		}
+		if p.ps.(*MemCachedStore) != s {
+			panic(fmt.Sprintf("persistent layer mismatch for memcached store %d", i)) // program bug
+		}
+		keys += len(p.mem) + len(p.stor)
+	}
+	if keys == 0 {
+		return 0
+	}
+
+	s.lock()
+	for _, p := range private {
+		s.MemoryStore.putChangeSet(p.mem, p.stor)
```

Comment:
Fun fact: p.mem and p.stor can be managed in separate routines here. No, we don't need to try it now (especially given the fact that `p.mem` is empty in AER store.